frontend/screens/ProfileScreen.tsx

```diff
@@ -0,0 +1,197 @@
+    // set Navigation Screen options leaving
+   useLayoutEffect(() => {
+    navigation.setOptions({
+      headerLargeTitle: true,
+      headerTitleStyle: {
+        fontSize: 30,
+        fontFamily: 'Raleway-Bold',
+        color:'white',
+      },
+      headerTitleAlign: "left",
+      headerTransparent: true,
+      headerShadowVisible: false,
+      headerLeft: () => (
+        <View>
+          <TouchableOpacity onPress={() => navigation.goBack()}
+          style={{paddingRight: 8}}>
+            <MaterialIcons
+              name="arrow-back-ios"
+              size={24}
+              color={'hsl(0, 0%, 100%)'}
+              style={{ paddingLeft: 10 }}
+            />
+          </TouchableOpacity>
+        </View>
+      ),
+    });
+  }, [navigation]);
+
+    return (
+        <View style={{ backgroundColor: 'white', flex: 1, }}>
+            <StatusBar translucent barStyle="light-content" backgroundColor="transparent"/>
+
+            <LinearGradient colors={['#843CDE', '#4A18DD', '#2E1181']} style={{flex: 1}}>
+                <SafeAreaView style={{alignItems: 'center', marginTop: headerHeight}}>
+                    <View style={{backgroundColor: 'white', marginTop: 20, width: 340, borderRadius: 15, padding: 25}}>
+                        <View style={styles.playlistInformation}>
+                            <Image source={profileImage} style={styles.image}/>
+                            <View style={styles.info}>
+                                <Text style={styles.spotifyName}>John Doe</Text>
+                                <TouchableOpacity>
+                                    <Text style={[styles.redirectText, styles.underline]}>Open on Spotify </Text>
+                                </TouchableOpacity>
+                            </View>
+                        </View>
+                        <View style={styles.divider}>
+                            <View style={styles.line} />
+                        </View>
+                        <View>
+                            <Text style={styles.playlistTitle}>Your Playlists</Text>
+                            <Text style={styles.redirectText}>List of Spotify playlists used to personalize results for you</Text>
+                        </View>
+                        <View style={styles.divider}>
+                            <View style={styles.line} />
+                        </View>
+                        <View>
+                            <Text style={styles.item}>Liked Songs</Text>
+                            <Text style={styles.item}>On Repeat</Text>
+                            <Text style={styles.item}>Time Capsule</Text>
+                            <Text style={styles.item}>Playlist Name 1</Text>
+                            <Text style={styles.item}>Playlist Name 2</Text>
+                        </View>
+                        <View style={styles.divider}>
+                            <View style={styles.line} />
+                        </View>
```

Comment:
Not the best code, but it works for now. If anyone has an alternative to the divider/line situation, please lmk